src/features/operator/Fee/FeeWithdraw/index.tsx

```diff
@@ -36,6 +51,26 @@ export const FeeWithdraw = (): JSX.Element => {
       ? 'N/A'
       : formatSatsToUnit(feeBalance?.totalBalance, lbtcUnit, LBTC_ASSET[network].asset_id, network);
 
+  const feeFiatAmount = useMemo(() => {
+    // display fiat value as 0 in case of api loading or error
+    if (isLoadingPrices || isErrorPrices) {
+      return <>{currency.symbol}0</>;
+    }
+    try {
+      const feeAMNT = Big(feeWithdrawAmount);
+      const feeLSAT = Number(formatLbtcUnitToSats(feeAMNT.toNumber(), lbtcUnit));
```

Comment:
Probably faster execution doing `Number(feeAMNT)` instead converting string to Big to Number